### DISCUSSION TITLE
Question about translating the game.

### DISCUSSION THREAD

**Discussion by GithubUser699** [colobot/colobot#1452] [Q&A]
Can I just edit the files (e.g. https://github.com/colobot/colobot-data/blob/master/levels/missions/chapter003/level003/po/de.po) or do I have to build it?
I could test if the translation is being displayed correctly by editing "/usr/share/games/colobot/levels/missions/chapter003/level003/help" for example, but cannot build...

**Comment by tomangelo2** [comment]
https://github.com/colobot/colobot/wiki/Working-with-translations

**Comment by emmax99x** [comment]
In my opinion it looks correct
